src/test/java/uk/gov/ons/ctp/integration/rhsvc/service/impl/RespondentDataServiceIT_Test.java

```diff
@@ -6,87 +6,92 @@
 import org.junit.Before;
 import org.junit.Ignore;
 import org.junit.Test;
-import uk.gov.ons.ctp.integration.rhsvc.domain.model.Address;
-import uk.gov.ons.ctp.integration.rhsvc.domain.model.CaseContext;
-import uk.gov.ons.ctp.integration.rhsvc.domain.model.UACContext;
+import uk.gov.ons.ctp.integration.rhsvc.domain.model.CollectionCase;
+import uk.gov.ons.ctp.integration.rhsvc.domain.model.UAC;
 import uk.gov.ons.ctp.integration.rhsvc.service.RespondentDataService;
 
+// import uk.gov.ons.ctp.integration.rhsvc.domain.model.Address;
+
 public class RespondentDataServiceIT_Test {
 
   private RespondentDataService cloud;
   private String caseId;
-  private String uacCode;
-  private UACContext uac;
-  private CaseContext caseContext;
+  private String uacHash;
+  private UAC uac;
+  private CollectionCase collectionCase;
 
   @Before
   public void setup() {
     cloud = new RespondentDataServiceImpl();
-    caseId = "123456789";
-    uacCode = "abcd-defg-1234";
-    String case_type = "H";
-    String questionaireId = "123456789";
-    String transactionId = "123456789";
-    String region = "EN";
-    String timestamp = "2019-02-01T08:15:30+00:00";
-    String addressLine1 = "Address line 1";
-    String addressLine2 = "Address line 2";
-    String city = "city";
-    String collection_exercise = "collection exercise";
-    String postCode = "PO155XX";
-    String uprn = "123456789012";
+    caseId = "c45de4dc-3c3b-11e9-b210-d663bd873d93";
+    uacHash = "72C84BA99D77EE766E9468A0DE36433A44888E5DEC4AFB84F8019777800B7364"; // SHA-256 hash of
+    // UAC
+    String caseRef = "10000000010";
+    String survey = "Census";
+    String collectionExerciseId = "n66de4dc-3c3b-11e9-b210-d663bd873d93";
+    String sampleUnitRef = "";
+    String addressLines = "Address lines";
+    String state = "actionable";
+    String actionableFrom = "2011-08-12T20:17:46.384Z";
+    //    String timestamp = "2019-02-01T08:15:30+00:00";
+    String active = "true";
+    String questionnaireId = "1110000009";
+    String caseType = "H";
+    String region = "E";
+
+    //    Address address = new Address();
+    //    address.setAddressLine1(addressLines);
 
-    uac = new UACContext();
-    uac.setUniversalAccessCode(uacCode);
+    collectionCase = new CollectionCase();
+    collectionCase.setId(caseId);
+    collectionCase.setCaseRef(caseRef);
+    collectionCase.setSurvey(survey);
+    collectionCase.setCollectionExerciseId(collectionExerciseId);
+    collectionCase.setSampleUnitRef(sampleUnitRef);
+    collectionCase.setAddress(addressLines);
+    collectionCase.setState(state);
+    collectionCase.setActionableFrom(actionableFrom);
+    //    collectionCase.setTimestamp(timestamp);
```

Comment:
Dead code should be deleted. Also lines 36, 42 & 43

Reply:
commented code now deleted